pvm/ts/src/engine/texasHoldem.ts

```
@@ -461,8 +475,35 @@
         // Handle non-player actions first (JOIN, LEAVE, DEAL)
         switch (action) {
             case NonPlayerActionType.JOIN:
-                this.join(address, amount!);
-                this.incrementTurnIndex();
+                // For JOIN action, extract seat from data string (0,2) - position 2 contains the seat
+                let preferredSeat: number | undefined = undefined;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)